br/pkg/streamhelper/advancer_daemon.go

```diff
@@ -48,6 +48,7 @@ func (c *CheckpointAdvancer) Name() string {
 
 func (c *CheckpointAdvancer) onStop() {
 	metrics.AdvancerOwner.Set(0.0)
+	metrics.LastCheckpoint.Reset()
```

Comment:
How about other metrics?  should also reset them?

Reply:
I think perhaps we don't need to reset them before some problems with them evaluated. 🤔

Reply:
can we put a comment here saying other metrics potentially need to be audit as well? And maybe a comment describing why do we need to reset checkpoint for this issue? I don't have context on this so just looking at the code I'm very clear what needs to be done. Thanks!